jekyll/_cci2/runner-config-reference-3.adoc

```diff
@@ -0,0 +1,327 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.3
```

Comment:
Per Jacob's comment [here](https://github.com/circleci/circleci-docs/pull/8342#discussion_r1330120312), we'll want this to read `Server v4.4+`.

Reply:
Yes let's remove, we've never tagged pages with versions that aren't available yet so. The tag should get added during the server 4.4 (or whichever) docs updates